### DISCUSSION TITLE
Changes from branches other than main or master do not count as contributions to the github page

### DISCUSSION THREAD

**Discussion by ervinanovlianti** [community/community#145281] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Why the changes from the other branches do not count as contribution to the github page, but we work in the branches?. Thank you

**Comment by AuroraEchos** [comment]
Hello, on GitHub, GitHub only records merging to the default branch (usually main or Master). If you are working on a branch, but these changes have not been incorporated into the main branch or the corresponding target branch, they may not show up on your contribution chart.
To fix this problem, you **make sure that the branch's changes are merged**, and after your changes are merged, you may have to wait up to 24 hours to see updates on the contribution chart.

**Comment by JorenVsyntra** [comment]
Hey everyone!
